### ISSUE TITLE
Improve Accessibilty of home page

### ISSUE THREAD

**Issue by Sky-De** [techy1999/BlogFrontend#107]
currently, accessibility is 89 percent, improve it to more than 95 percent

**Current lack of accessibility**

![Screenshot 2024-05-01 062728](https://github.com/techy1999/BlogFrontend/assets/79264045/ffe700de-7270-4b46-b347-16fcb3373c68)

**Comment by Sky-De** [comment]
@techy1999 please assign this to me, i will create PR soon, thanks

**Comment by techy1999** [comment]
@Sky-De Assigned you can work on it.

**Comment by techy1999** [comment]
> @Sky-De Assigned you can work on it.
